src/pages/SignupPage/styles.js

```diff
@@ -0,0 +1,96 @@
+import { css } from 'react-emotion';
+
```

Comment:
Please use the styles from login page (buttons and card) instead of re-creating this file.